### PR TITLE
Use markdown for news

### PR DESCRIPTION
**Pull request by Chocobo1**
Writing/reading in HTML is error-prone, I added parsedown lib to allow writing in markdown.

@sledgehammer999 
Hope you don't mind this change.

### CODE REVIEW

**Comment by sledgehammer999** [comment]
Don't merge this. I want to review this at a later time.

**Comment by Chocobo1** [comment]
Rebased on master

File: news.php [hunk 1]
```diff
@@ -6,6 +6,6 @@
 require_once('lib/helpers.php');
 readMarkdown('news.md');
 
-require_once('news.inc');
```
Comment (sledgehammer999) [inline news.php]: Leave a comment here explaining what is happening. Explain that at date X we converted to writing news in Markdown and didn't bother converting the whole old news to it.

**Comment by sledgehammer999** [comment]
Well, I haven't tested it on how it renders/works since I don't have a webserver ready. But anyhow, I trust you that it does.

Anyway, you're free by me to merge, addressing the one small issue.

File: news.php (hunk 1 above)
Comment (Chocobo1) [inline news.php]: Done.

**Pull request merged**